13LessonThirteen/homework/css/style.css

```diff
@@ -0,0 +1,50 @@
+header {
+  background-image: url(http://bone.themanoid.com/images/office/03.jpg);
+  background-repeat: no-repeat;
+  height: 600px;
+  width: 100%;
+  background-position: center;
+  text-align: center;
+  padding-top: 200px;
+  font-size: 80px;
+}
+.topleft {
+  height: 257px;
+  width: 349px;
+}
+.grid {
+  padding-left: 100px;
+}
+.right {
+  height:523px;
+  padding-top: 10px;
+}
+.midbottom {
+  padding-top: 10px;
+}
+.bottomleft {
+  padding-top: 10px;
+}
+.topleft {
+  padding-top: 10px;
+}
+.midtop {
+  padding-top: 10px;
+  height: 257px;
+  width: 349px;
+}
+.silver {
+  background-color: white;
+  border: 2px solid white;
+  text-align: center;
+}
+.container {
+  background-color: #EFF1F0;
+}
+.gold {
+  background-color: #D7F8E5;
+  text-align: center;
```

Comment:
tip: bootstrap has a class called `text-center` that you can add to your HTML elements - it will center the text in that element and all its child elements!